lib/src/visualisation.dart

```diff
@@ -6,7 +6,7 @@ class Visualisation {
   ColorTransformFunction _colorTransform;
   List<int> xs = [];
   List<int> ys = [];
-  List<num> values = [];
+  List<Map> dataList = [];
```

Comment:
datas would be more consistent, and would have the additional entertainment of being bad latin.